### PR TITLE
task 5 logging and error handling

### PR DESCRIPTION
**Pull request by Boilingwaterr**

### CODE REVIEW

File: src/controllers/user.controller.ts [hunk 1]
```diff
@@ -31,7 +32,8 @@ export const getAllUsers: RequestHandler = async (
     );
 
     if (suggestedUser) {
-      return res.json(suggestedUser);
+      res.json(suggestedUser);
+      return;
```
Comment (masiacra) [inline src/controllers/user.controller.ts:36]: let's use next() after res.json(suggestedUser) in other case another middleware never get control

File: src/controllers/user.controller.ts [hunk 2]
```diff
@@ -85,7 +89,7 @@ export const updateUser: RequestHandler = async (req, res, next) => {
         return res.json({ id });
```
Comment (masiacra) [inline src/controllers/user.controller.ts]: same here abot use next() and everywhere where it is possible

Comment (Boilingwaterr) [reply src/controllers/user.controller.ts]: added next (), thanks for explanations

File: src/controllers/user.controller.ts (hunk 1 above)
Comment (Boilingwaterr) [reply src/controllers/user.controller.ts:36]: added next (), thanks for explanations

**Pull request merged**